src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -32,6 +33,13 @@ const GovernanceReviewTeam = () => {
     (state: AppState) => state.businessCase.form
   );
 
+  const userGroups = useSelector((state: AppState) => state.auth.groups);
+  const authorizedToView = userGroups.includes('EASI_D_GOVTEAM');
```

Comment:
Should we store these group names in an object somewhere? If so, where?

Reply:
FYI, I believe you also want to check for the job code that is used in Prod as well: `EASI_P_GOVTEAM`... 
(When I initially implemented on the back-end, I made life more complicated by looking for only one at a time per environment, but eventually I realized that because of how they are configured those two job codes will never actually co-exist in one environment. So you can functionally check for both, and having either one can be used as a rule-in for GRT powers.)

Reply:
Good catch, Nelz! Thanks.